src/Components/Medicine/MedicineAdministrationSheet/AdministrationTableRow.tsx

```diff
@@ -222,7 +222,7 @@ export default function MedicineAdministrationTableRow({
                 </p>
               )}
 
-              <p>
+              <p className="max-w-[110px] truncate">
```

Comment:
avoid hardcoding arbitrary pixel values. This will lead to different results in different browser zoom settings

Reply:
6 rem is working fine... Shall i add that? What do you think?

Reply:
Yup